site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -236,7 +236,14 @@ The following tables lists the configurable parameters of the Agones chart and t
 
 | Parameter                                           | Description                                                                                     | Default                |
 | --------------------------------------------------- | ----------------------------------------------------------------------------------------------- | ---------------------- |
-|                                                     |                                                                                                 |                        |             
+| `agones.controller.annotations`                     | Controller [annotations][annotations] settings for pod assignment                               | `{}`                   |
+| `agones.allocator.annotations`                      | Allocator [annotations][annotations] settings for pod assignment                                | `{}`                   |
+| `agones.ping.annotations`                           | Ping [annotations][annotations] settings for pod assignment                                     | `{}`                   |
+| `agones.ping.http.annotations`                      | [Annotations][annotations] for ping http service                                                | `{}`                   |
```

Comment:
```suggestion
| `agones.ping.http.annotations`                      | [Annotations][annotations] added to the Agones ping http service                                             | `{}`                   |
```

Minor tweaks for just some extra clarity based on Robbie's suggestions. Just to make things extra clear.

Reply:
Fixed.